site/src/common/blocks/AccordionItemBlock.tsx

```diff
@@ -31,21 +31,18 @@ type AccordionItemBlockProps = PropsWithData<AccordionItemBlockData> & {
 
 export const AccordionItemBlock = withPreview(
     ({ data: { title, content }, isExpanded, onChange }: AccordionItemBlockProps) => {
-        const intl = useIntl();
-
-        const ariaLabelText = isExpanded
-            ? intl.formatMessage({ id: "accordionBlock.ariaLabel.expanded", defaultMessage: "Collapse accordion item" })
-            : intl.formatMessage({ id: "accordionBlock.ariaLabel.collapsed", defaultMessage: "Expand accordion item" });
+        const headlineId = useId();
+        const contentId = useId();
 
         return (
             <>
-                <TitleWrapper onClick={() => onChange()} aria-label={ariaLabelText}>
+                <TitleWrapper id={headlineId} onClick={() => onChange()} aria-label={title} aria-expanded={isExpanded} aria-controls={contentId}>
```

Comment:
If the plain text already is the content, there is no need to additionally set the aria-label with the same content. This should can be removed

```suggestion
                <TitleWrapper id={headlineId} onClick={() => onChange()} aria-expanded={isExpanded} aria-controls={contentId}>
```